schema/metaschema/metaschema-module-metaschema.xml

```diff
@@ -1027,6 +1027,7 @@
         <constraint>
             <allowed-values>
                 <enum value="CRITICAL">A violation of the constraint represents a serious fault in the content that will prevent typical use of the content.</enum>
+                <enum value="DEBUG">A violation of the constraint represents a fault in the content that warrants review by a developer when performing model or tool development.</enum>
```

Comment:
Move this to after INFORMATIONAL, since these are listed in order of severity?

Reply:
Relocated as requested in https://github.com/usnistgov/metaschema/commit/6ac24990042fded121843e8ee9e51a4b988e71bb.